.github/CODEOWNERS

```diff
@@ -734,6 +734,9 @@
 # ServiceLabel: %Network - Private Link
 # ServiceOwners:                                                   @privlinksuppgithub
 
+# PRLabel: %Network - Network Cloud
```

Comment:
Please move this to the management library section after L946 and follow the patterns there, updating the path and label entries and including designating service owners for support purposes.